tests/unit/handler.unitTest.ts

```diff
@@ -161,6 +161,31 @@ describe("The lambda function handler", () => {
         expect(TechRecordsService.prototype.updateTechRecordStatusCode).toHaveBeenCalled();
       });
 
+      it("should call the update-eu-vehicle-category function with correct event payload", async () => {
```

Comment:
test doesn't actually check the payload

Reply:
if you name the mock before assigning it to the prototype, you can use mockName.mock.calls[0].args to get the calling payload

Reply:
used mockName.mock.calls[0][0] and mockName.mock.calls[0][1]